CHANGELOG.md

```diff
@@ -1,122 +1,124 @@
-- Force activation of RPC ring buffer when load balancing is enabled, to allow RPC handover when authority changes
-- Fixed a race where a client leaving the deployment could leave its actor behind on the server, to be cleaned up after a long timeout.
-- Fixed crash caused by state persisting across a transition from one deployment to another in SpatialGameInstance.
-- Fixed crash when starting + stopping PIE multiple times.
-- Fixed crash when shadow data was uninitialized when resolving unresolved objects.
+- The GDK now tracks properties that contain references to replicated Actors, so that it can resolve them again if the Actor that they reference moves out of and back into relevance.
+- PIE sessions no longer occasionally fail to start due to missing schema for the `SpatialDebugger` Blueprint.
+- We've fixed an issue where a newly-created subobject had empty state when `RepNotify` was called for a property pointing to that subobject.
+- We've fixed an issue where deleted, initially dormant startup Actors would still be present on other worker instances.
+- We now force-activate the RPC ring buffer when load balancing is enabled, to allow RPC handover when authority changes.
+- We've fixed a race condition where a client that was leaving the deployment could leave its Actor behind on the server, to be cleaned up after a long timeout.
+- We've fixed a crash that was caused by state persisting across a transition from one deployment to another in `SpatialGameInstance`.
+- The GDK no longer crashes when you start and stop PIE clients multiple times.
+- The GDK no longer crashes when shadow data is uninitialized when resolving unresolved objects.
 - Fixed sending component RPCs on a recently created actor.
-- Fix problem where load balanced cloud deploys could fail to start while under heavy load.
-- Fix to avoid using packages still being processed in the async loading thread.
-- Fixed a bug when running GDK setup scripts fail to unzip dependencies sometimes.
+- Load-balanced cloud deployments no longer fail to start while under heavy load.
+- We now avoid using packages that are still being processed in the async loading thread.
+- Running GDK setup scripts no longer occasionally fails to unzip dependencies.
+
+### Internal:
+Features listed in this section are not ready to use. However, in the spirit of open development, we record every change that we make to the GDK.
+
+- We've enabled the SpatialOS toolbar for MacOS.
+- We've added support for Android.
+- `SpatialDebugger` worker regions are now cuboids rather than planes, and can have their `WorkerRegionVerticalScale` adjusted via a setting in the `SpatialDebugger`.
+- We've added an `AuthorityIntent` component, a `VirtualWorkerTranslation` component, and a partial framework. We'll use these in the future to control load balancing.
+- Load balancing strategies and locking strategies can be set per-level using `SpatialWorldSettings`.
```

Comment:
I couldn’t find “SpatialWorldSettings” in the docs. Is there definitely something with this name? [EDIT: Ernie has confirmed that this is internal only, so it might be OK.]

Reply:
`SpatialWorldSettings` is a class in the GDK that users can inherit from. Developer concept.